tests/subsys/logging/log_stack/src/main.c

```diff
@@ -133,9 +133,9 @@ ZTEST_SUITE(test_log_stack, NULL, NULL, NULL, NULL, after);
 #define HEXDUMP_USAGE 528
 #define MORE_ARGS_USAGE 560
 #elif defined(CONFIG_LOG_MODE_IMMEDIATE) && defined(CONFIG_NO_OPTIMIZATIONS)
-#define SIMPLE_USAGE 1520
-#define HEXDUMP_USAGE 1504
-#define MORE_ARGS_USAGE 1584
+#define SIMPLE_USAGE 1552
+#define HEXDUMP_USAGE 1536
+#define MORE_ARGS_USAGE 1616
```

Comment:
> The amount of stack used has changed for some reason (toolchain updates?),
use the amounts reported by the test when it failed.

This is likely from the main branch (same failure observed there too) -- we have been discussing whether this test should be disabled because the stack usage tends to change from time to time and it does not make sense to make the CI fail because of this (FYI @nordic-krch).

For now, this patch should do.